### ISSUE TITLE
How to setup for docker and or vendor/bundle?

### ISSUE THREAD

**Issue by mhenrixon** [SublimeLinter/SublimeLinter-rubocop#77]
Has anyone successfully setup the rubocop linter for projects running exclusively inside docker? Currently, I'm having all gems installed inside `vendor/bundle` in the hopes of making them more accessible.

That said, all gems are installed inside a Linux docker container and might not be usable from outside docker (my host is a Mac beast).

Just curious if anyone has finally solved this issue. Would be nice to have integration with rubocop inside the docker container somehow.

**Comment by joshmfrankel** [comment]
I've had decent success interacting with rubocop via docker-compose by first creating a shell executable file

touch docker-rubocop

```
#!/bin/bash
docker-compose run web bundle exec rubocop $@
```

Then just specify the absolute path to the executable within your SublimeLinter settings file

```
    "rubocop": {
      "executable": "~/Development/project_folder/docker-rubocop"
    }
```

**Comment by kaste** [comment]
Shouldn't

```
   "executable": ["docker-compose", "run", "--rm", "web", "bundle", "exec", "rubocop"]
```

then also work, 🤔?

**Comment by joshmfrankel** [comment]
Good point @kaste! Seems reasonable to assume it would. Can't remember why I didn't try that first 🤔
